JavaScript/BOMExample(location).js

```diff
@@ -0,0 +1,4 @@
+alert(location.href); 
```

Comment:
1. Avoid braces in file names.
2. File naming convention : There is no official  convention for naming JavaScript files but people usually follow the convention used by jQuery. Lower case words seperated by hypen or full stop. `bom_location_example.js`

Reply:
Got it.Fixed.